src/shared/verbs.c

```diff
@@ -45,10 +45,23 @@ bool running_in_chroot_or_offline(void) {
         return r > 0;
 }
 
+const Verb* verbs_find_verb(const char *name, const Verb verbs[]) {
+        for (unsigned i = 0;; i++) {
+                /* At the end of the list? */
+                if (!verbs[i].dispatch)
+                        break;
+
+                if (streq_ptr(name, verbs[i].verb) ||
+                    (!name && verbs[i].flags & VERB_DEFAULT))
```

Comment:
hmm, && and & in one expresion is a bit hard to read. consider using () or FLAGS_SET()